pydoctor/test/test_astbuilder.py

```diff
@@ -452,7 +452,7 @@ class Processor:
     f = P.contents['clientFactory']
     assert unwrap(f.parsed_docstring) == """Callable that returns a client."""
     assert f.privacyClass is model.PrivacyClass.VISIBLE
-    assert f.kind == 'Instance Variable'
+    assert f.kind == model.KindClass.INSTANCE_VARIABLE
```

Comment:
You can use `is` instead of `==` here as well. Performance is not that important for the tests, but it's good for consistency.

Reply:
What should I understand  here ? Please don’t tell me that I should replace All double equals with is when possible ...

Reply:
You could do an automated search-and-replace to replace the obvious ones; if there are any left after that, it's not really a problem.